Towny/src/main/java/com/palmergames/bukkit/util/ItemLists.java

```diff
@@ -4,7 +4,6 @@
 import java.lang.reflect.Modifier;
 import java.util.Arrays;
 import java.util.Collection;
-import java.util.List;

```

Comment:
Do not remove this import.

Reply:
Actually ignore this. Somehow my ide didn't show this as being unused.

Reply:
it was unused, but sure